pkg/ddevapp/config.go

```diff
@@ -463,6 +463,10 @@ func (app *DdevApp) RenderComposeYAML() (string, error) {
 	if util.ArrayContainsString(app.OmitContainers, "dba") {
 		includeDBA = ""
 	}
+	var includeSSHAgent = "includeItByDefault"
```

Comment:
My bet is you can school me in a cleaner way to do this in the template @andrewfrench . For some reason I just hate golang templating and haven't achieved much competency.